src/main/java/org/fundacionjala/pivotal/objects/DashBoard.java

```diff
@@ -0,0 +1,58 @@
+package org.fundacionjala.pivotal.objects;
+
+import org.fundacionjala.pivotal.common.Base;
+import org.fundacionjala.pivotal.common.CommonMethods;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+
+/**
+ * Class to Dashboard for return of any page to Dashboard.
+ */
+public class DashBoard extends Base {
+
+    @FindBy(how = How.CSS, using = "#elm-root  a.SettingsIcon__cog.projectTileHeader__hoverable")
+    private WebElement buttonEditProject;
+
+    @FindBy(how = How.ID, using = "projects-search-bar")
+    private WebElement searchProjectButton;
+
+    @FindBy(how = How.XPATH, using = "//*[contains(@href,'/dashboard')] ")
+    private WebElement buttonDashborad;
+
+    /**
+     * Set Button DashBoard.
+     */
+    public void setButtonDashborad() {
+        ((JavascriptExecutor) driver).executeScript("arguments[0].click();", buttonDashborad);
+        //CommonMethods.clickWebElement(buttonDashborad);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)